config/indexer.go

```diff
@@ -34,6 +36,7 @@ func NewIndexer() Indexer {
 		ShutdownTimeout:     Duration(10 * time.Second),
 		ValueStoreDir:       "valuestore",
 		ValueStoreType:      "sth",
+		STHBits:             24,
```

Comment:
Add comment about changing this value for an existing datastore and its consequences?